Added flag to cockroach demo for disabling rangefeeds (#14636)
cockroachdb/docs: v22.1/cockroach-demo.md

Flag | Description
-----|------------

> `--auto-enable-rangefeeds` | Override the default behavior of `cockroach demo`, which has rangefeeds enabled on startup. Use `--auto-enable-rangefeeds=false` to disable rangefeeds — this will improve performance. To run a [changefeed](create-and-configure-changefeeds.html) on a demo cluster, you must enable rangefeeds. See [Enable rangefeeds](use-changefeeds.html#enable-rangefeeds) for more detail. <br><br>**Default:** `true`

Reviewer: nit: Consider mentioning "rangefeeds required for changefeeds" a bit earlier, maybe something like:

```suggestion
`--auto-enable-rangefeeds` | Override the default behavior of `cockroach demo`, which has rangefeeds enabled on startup.  If you do not need to use [changefeeds](create-and-configure-changefeeds.html) with your demo cluster, use `--auto-enable-rangefeeds=false` to disable rangefeeds and improve performance. See [Enable rangefeeds](use-changefeeds.html#enable-rangefeeds) for more detail. <br><br>**Default:** `true`
```
